modules/websocket/emws_server.cpp

```diff
@@ -32,12 +32,16 @@
 
 #include "emws_server.h"
 #include "core/os/os.h"
+#include "core/dictionary.h"
 
 Error EMWSServer::listen(int p_port, Vector<String> p_protocols, bool gd_mp_api) {
 
 	return FAILED;
 }
 
+void EMWSServer::set_headers(Dictionary headers) {
```

Comment:
I'd prefer using Vector<String> (like we do in `HTTPClient`).
Does it make sense to change the headers once the server is started? Should we just make this an extra parameter to `listen`?
In any case, I think the name should be either something like `extra_headers` or `custom_headers`.

Reply:
Actually I thought that but I thought common usage is `Dictionary` from GDScript. I thought also passing parameter to `listen()` but is it a better idea? Also we can use Array instead of Dictionary like `[["Header", "Value"], ["Header", "Value"]]`.

Reply:
> Also we can use Array instead of Dictionary like `[["Header", "Value"], ["Header", "Value"]]`.

No, as mentioned, I would use `Vector<String>` (i.e. `PackedStringArray` in GDScript): `PackedStringArray(["header: value;", "header2: value2;"])` like in [`HTTPClient.request`](https://docs.godotengine.org/en/latest/classes/class_httpclient.html#class-httpclient-method-request)

Reply:
Ok. Actually we had to escape quotes for value with `["header", "value"]` or `{"header": "value"}` syntax but it is a user issue with `["header: value", ...]` usage. I will switch it to Vector.